coalib/processes/Processing.py

```diff
@@ -291,7 +292,12 @@ def get_file_dict(filename_list, log_printer=None, allow_raw_files=False):
     return file_dict
 
 
-def filter_raising_callables(it, exception, *args, debug=False, **kwargs):
+def filter_raising_callables(it,
+                             exception,
+                             *args,
+                             debug=False,
+                             debug_bears=False,
+                             **kwargs):
```

Comment:
do you really need to modify this function? This one is very generic and has the only purpose to filter out return values, whose function that produces them raises an error.

Reply:
This was actually a reason because of which I wanted to review this part of code, because this is where Bear is getting instantiated and also current test of function is thing which I can't make it to work because it will get unexpected kwarg `debug_flag`

Reply:
I mean pass it via the already existing `args` parameter ;)